client/src/components/Payment.jsx

```diff
@@ -52,7 +65,7 @@ class Payment extends React.Component {
   payUser() {
     let payment = {
       payerId: this.props.payerId,
-      payeeUsername: this.state.payeeUsername,
+      payeeUsername: !this.state.payeeUsername ? this.props.payeeUsername : this.state.payeeUsername,
       amount: this.state.amount,
```

Comment:
@jGlass314 - you can accomplish this same logic with
`this.state.payeeUsername || this.props.payeeUsername`
It's the equivalent of 
`!this.state.payeeUsername ? this.props.payeeUsername : this.state.payeeUsername`